bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/DataClientVeneerApi.java

```diff
@@ -229,8 +249,97 @@ protected void onCompleteImpl() {
   }
 
   /** wraps {@link ServerStream} onto HBase {@link ResultScanner}. */
-  private static class RowResultScanner extends AbstractClientScanner {
+  static class PaginatedRowResultScanner extends AbstractClientScanner {
```

Comment:
This definitely needs more doc and an explanation how its meant to be used. The audience of the documentation should be a client developer

Reply:
Done